worker/__init__.py

```diff
@@ -73,36 +71,57 @@ def fetch_events(self, lat, lon, rad, **params):
     # Add additional parameters.
     data.update(params)
 
-    # Request.
-    result = eb.event_search(**data)
+    # Find out how many pages to collect.
+    pagination = eb.event_search(**data)['pagination']
+    num_pages = pagination['page_count']
+    to_collect = min(num_pages, config.EVENTBRITE_EVENT_PAGES)
 
     # Produce a new task to load every event.
     with session_scope() as db_session:
-        for event in result['events']:
-            venue = eb.get('/venues/{}'.format(event['venue_id']))
-            new_event = dict(
-                id=event['id'],
-                title=event['name']['text'],
-                start_time=event['start']['utc'],
-                end_time=event['end']['utc'],
-                start_time_local=event['start']['local'],
-                end_time_local=event['end']['local'],
-                timezone=event['start']['timezone'],
-                capacity=event['capacity'],
-                location=WKTElement('POINT({} {})'.format(
-                    venue['longitude'],
-                    venue['latitude']
-                )),
-                logo=event['logo'],
-                url=event['url'],
-                description_text=event['description']['text'],
-                description_html=event['description']['html'],
-                is_free=event['is_free'],
-                online_event=is_online(event, venue)
+
+        # Collect several pages.
+        logger.info('Collecting {} pages'.format(num_pages))
+        logger.info(pagination)
+        for page in range(1, to_collect+1):
+
+            # Collect this page.
+            data.update({'page': page})
+            result = eb.event_search(**data)
+            logger.info('{} of {} pages: collected {} events'.format(
+                num_pages,
+                result['pagination']['page_size']
+                )
             )
 
-            # Load event into database.
-            load_event(new_event, db_session)
+            for event in result['events']:
+
+                # Collect venue for lat/lon.
+                venue = eb.get('/venues/{}'.format(event['venue_id']))
+
+                # Build a payload with each event.
+                new_event = dict(
+                    id=event['id'],
+                    title=event['name']['text'],
+                    start_time=event['start']['utc'],
+                    end_time=event['end']['utc'],
+                    start_time_local=event['start']['local'],
+                    end_time_local=event['end']['local'],
+                    timezone=event['start']['timezone'],
+                    capacity=event['capacity'],
+                    location=WKTElement('POINT({} {})'.format(
+                        venue['longitude'],
+                        venue['latitude']
+                    )),
+                    logo=event['logo'],
```

Comment:
[no change req'd, just curious] what is the format of this column?

Reply:
Unsure how much of this @jordanschalm wants/needs.

```json
"logo": {
            "crop_mask": null,
            "original": {
                "url": "https://img.evbuc.com/https%3A%2F%2Fcdn.evbuc.com%2Fimages%2F17710414%2F17858050391%2F1%2Foriginal.png?s=514fe67d559e43224d1cd7ebbaef3bd4",
                "width": null,
                "height": null
            },
            "id": "17710414",
            "url": "https://img.evbuc.com/https%3A%2F%2Fcdn.evbuc.com%2Fimages%2F17710414%2F17858050391%2F1%2Foriginal.png?h=200&w=450&s=839f6bebd1c8e09e766094ba4116f334",
            "aspect_ratio": "4.945054945054945054945054945",
            "edge_color": null,
            "edge_color_set": true
        }
```